### DISCUSSION TITLE
11 Star Medals, and possibly beyond

### DISCUSSION THREAD

**Discussion by sinc0s** [ppy/osu#27785] [Ideas] [answered]
Simple idea. The skill of top players is teaching a point where the current amount of medals is not enough. We have had points in the past where FCing an 11 star map was shown to be possible, and as of this week, it very nearly happened. So, it would make sense for medals to be added for this, and possibly for even higher stars as well, as it's pretty clear that achieving a 12 star FC will happen at some point as well.

If there is a desire to keep the 10 star FC medal as the highest due to the clean double digit cap, and the fact that higher star medals would be achieved even less, that's understandable. I would simply like to use this discussion to gather opinions.

**Comment by Walavouchey** [comment, marked as answer]
tracked in https://github.com/ppy/osu-web/issues/8350
